### PR TITLE
Clean up and minor design tweak

### PR DESCRIPTION
**Pull request by jina**
- Add `abbr` elements.
- Move SEO component to templates folder for better organization
- Remove unused violet token set
- Remove unused clearfix
- Fix media object
- Make logo smaller
- Clean up unnecessary default props

### CODE REVIEW

**Comment by jina** [comment]
Looking into errors

**Comment by jina** [comment]
🤷‍♀️ I don't know why it's failing. No errors at all when I run locally,

**Comment by kaelig** [comment]
I also ran into this build issue and it should be fixed in #65.

**Comment by kaelig** [comment]
👋 I just merged #65 – I updated this branch to see if it fixes the issue

**Comment by jina** [comment]
tests still seem to be failing. :(

File: website/src/dependencies/_mixins.scss [hunk 1]
```diff
@@ -35,18 +35,6 @@
   }
 }
 
-@mixin clearfix {
```
Comment (kaelig) [inline website/src/dependencies/_mixins.scss:38]: 🔥

Comment (jina) [reply website/src/dependencies/_mixins.scss:38]: i was using it for a non float issue actually then decided to use a different method

**Pull request merged**